File: JAVA/Program21.java
/*

Accept number from user and display below pattern.
Input : 4
Output : # 1 * # 2 * # 3 * # 4 *

*/


import java.util.*;

class Demo21
{
	 public void Display(int iNo)
	 {
	 	int i = 0;

         System.out.print("\n");

	 	for(i = 1; i <= iNo; i++)
	 	{
	 		System.out.print("# "+i+" *"+"\t");
	 		
	 	}
	 }

}

class Program21
{
	public static void main(String[] args) 
	{
	   Scanner sobj = new Scanner(System.in);
	   Demo21 dobj = new Demo21();

	   int iVal = 0;
       
       System.out.print("\nEnter Number : ");
       iVal = sobj.nextInt();

	   dobj.Display(iVal);

	   System.out.print("\n");

	}
}

File: C#/Program21.cs
/*

Write a program which accept number from user and count frequency of 2 in it.
Input : 2395 
Output : 1
Input : 1018
Output : 0

*/


using System;

class Demo{
 
    public int CountFrequency(int iNo){
     
        int iDigit  = 0;
        int iCnt = 0;

        while(iNo != 0){

            iDigit = iNo % 10;

              if(iDigit  == 2)
              {
                 iCnt++;
              }

            iNo = iNo / 10;

        }

       return iCnt;

    }

}

class program21{
       
     public static void Main(string[] args){

            int iNo = Convert.ToInt32(Console.ReadLine());
            
            Demo dobj = new Demo();

            int iRet = dobj.CountFrequency(iNo);

            
            Console.WriteLine(iRet);
            

     }
     
}